civihr_default_theme/template.php

```diff
@@ -167,9 +167,12 @@ function civihr_default_theme_form_element_label($variables) {
  * @return array
  */
 function bootstrapable_fields($fields_structure) {
-  return array_filter($fields_structure, function($value, $key) {
-    return substr($key, 0, 1) <> '#' && !in_array($value['#type'], ['hidden']);
-  }, ARRAY_FILTER_USE_BOTH);
+  foreach($fields_structure as $key => $value)  {
+    if (!(substr($key, 0, 1) <> '#' && !in_array($value['#type'], ['hidden'])))  {
+      unset($fields_structure[$key]);
```

Comment:
Question: i believe that [array_filter](http://php.net/manual/en/function.array-filter.php) returns a *new* array (you might want to confirm it), but here you're modifying the existing one. Could you make sure that the original `$fields_structure` remains unchanged?